### PR TITLE
Add package_dnsmasq_removed rule

### PR DESCRIPTION
**Pull request by cortesana**
#### Description:

The new package_dnsmasq_removed rule is created in order to meet the following CIS requirement for RHEL9:
- 2.2.14 - Ensure dnsmasq is not installed. (Automated)

#### Rationale:
Unless the system is specifically designated to act as a DNS caching, DNS forwarding and/or DHCP server, the package should be removed to reduce the potential attack surface.

### CODE REVIEW

File: linux_os/guide/services/dns/package_dnsmasq_removed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+description: |-
+    dnsmasq is a lightweight tool that provides DNS caching, DNS forwarding and
+    DHCP (Dynamic Host Configuration Protocol) services.
+    <br />
+    {{{ describe_package_remove(package="dnsmasq") }}}
+
+rationale: |-
+    Unless a system is specifically designated to act as a DNS
+    caching, DNS forwarding and/or DHCP server, it is recommended that the
+    package be removed to reduce the potential attack surface
```
Comment (Mab879) [inline linux_os/guide/services/dns/package_dnsmasq_removed/rule.yml:16]:
```suggestion
    package be removed to reduce the potential attack surface.
```
Nitpick.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR! I found a few items that need to be addressed. Let me know if you have any questions.

File: controls/cis_rhel9.yml [hunk 2]
```diff
@@ -823,7 +823,10 @@ controls:
     levels:
       - l1_server
       - l1_workstation
-    status: planned
+    status: automated
+    rules:
+      - package-dnsmasq-removed
```
Comment (marcusburghardt) [inline controls/cis_rhel9.yml:828]:
```suggestion
      - package_dnsmasq_removed
```

Comment (marcusburghardt) [reply controls/cis_rhel9.yml:828]: The rule id is basically the folder name where the respective `rule.yml` is located. I believe you already know it, but in any case. ; )

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Thanks for this rule @cortesana . There is only a possible typo in the rule id.

File: controls/cis_rhel9.yml (hunk 2 above)
Comment (cortesana) [reply controls/cis_rhel9.yml]: Thank you for catching it :)

**Review by marcusburghardt — APPROVED** [review]
Thanks @cortesana

**Comment by marcusburghardt** [comment]
The Automatus CS8 failure is expected since the rule is restricted to `rhel9`.
The PR LGTM but we should wait @Mab879 to double-check since he is currently assigned to this PR.

**Review by Mab879 — APPROVED** [review]
LGTM, was just doing some testing on my end.

**Comment by jusgomezpa** [comment]
Is it posible to add this rule into RHEL 7 and 8 guides? 

In RHEL 7 and 8 the rule is 2.2.5?

**Pull request merged**